Gitflow_Script.sh

```diff
@@ -0,0 +1,39 @@
+!/bin/bash
+
+#Script for Gitflow excercise
+
+#Alfredo H. Locht ahlocht@dons.usfca.edu
+
+#13Sep2017
+
+cd 2017-09-11_git_exercise
```

Comment:
No need to do this - if you leave it out, then you can run this script to set up a similar project anytime you need it.